packages/pods-core/src/builtin/NotionPod.ts

```diff
@@ -0,0 +1,156 @@
+import { DendronError, ERROR_SEVERITY, NoteProps } from "@dendronhq/common-all";
+import { markdownToBlocks } from "@instantish/martian";
+import type {
+  Page,
+  TitlePropertyValue,
+} from "@notionhq/client/build/src/api-types";
+import { Client } from "@notionhq/client";
+import _ from "lodash";
+import { ExportPod, ExportPodPlantOpts, ExportPodConfig } from "../basev3";
+import { JSONSchemaType } from "ajv";
+import { NotionUtilMethods, PodUtils } from "../utils";
+import { RateLimiter } from "limiter";
+
+const ID = "dendron.notion";
+
+// Allow 3 req/sec. Also understands 'hour', 'minute', 'day', or a no. of ms
+// @ts-ignore
+const limiter = new RateLimiter({ tokensPerInterval: 3, interval: "second" });
+
+type NotionExportPodCustomOpts = {
+  apiKey: string;
+  vault: string;
+};
+
+export type NotionExportConfig = ExportPodConfig & NotionExportPodCustomOpts;
+
+export class NotionExportPod extends ExportPod<NotionExportConfig> {
+  static id: string = ID;
+  static description: string = "export notes to notion";
+
+  get config(): JSONSchemaType<NotionExportConfig> {
+    return PodUtils.createExportConfig({
+      required: ["apiKey", "vault"],
+      properties: {
+        apiKey: {
+          type: "string",
+          description: "Api key for Notion",
+        },
+        vault: {
+          type: "string",
+          description: "vault to export from",
+        },
+      },
+    }) as JSONSchemaType<NotionExportConfig>;
+  }
+
+  /**
+   * Method to create pages in Notion
+   */
+  createPagesInNotion = async (blockPagesArray: any, notion: Client) => {
+    // rate limiter method
+    const sendRequest = async () => {
```

Comment:
any reason to wrap this in a const and call it? as opposed to just having the block of code on the inside